nearcore/src/lib.rs

```diff
@@ -498,3 +499,88 @@ pub fn start_with_config_and_synchronization(
         arbiters: vec![client_arbiter_handle, arbiter.handle()],
     })
 }
+
+pub fn recompress_storage(home_dir: &Path, dst_dir: &Path) -> anyhow::Result<()> {
+    use strum::{EnumCount, IntoEnumIterator};
+
+    // We’re configuring each RocksDB to use 512 file descriptors.  Make sure we
+    // can open that many by ensuring nofile limit is large enough to give us
+    // some room to spare over 1024 file descriptors.
```

Comment:
Probably it makes sense to add the command to the helper info of the command: `ulimit -n 1024`